docs/low-code-spark/gems/custom/rest-api-enrich.md

```diff
@@ -0,0 +1,119 @@
+| verify          | Either a boolean, in which case it controls whether we verify the server’s TLS certificate eg: `true` or `false` or a string, in which case it must be a path to a CA bundle to use. Defaults to True. eg: `dbfs:/path-to-file` | false    | true    |
+| stream          | if False, the response content will be immediately downloaded. eg: `true` or `false`                                                                                                                                            | false    |         |
+| cert            | if String, path to ssl client cert file (.pem). eg. `dbfs:/path-to-file`. If Tuple, (‘cert’, ‘key’) pair. eg: `cert:key`.                                                                                                       | false    |         |
+| parse content   | Parse content as JSON (to make the schema available, please enable `custom schema` and click `infer from cluster` at the bottom left in the output tab)                                                                         | false    | false   |
+
+:::info
+
+1. To store sensitive information like API key (headers), auth etc., `Databricks secrets` can be used as shown in above
+   example.
+2. If the number of rows are very huge for which api calls needs to be made, it's better to provide `await time` in the
+   `advanced tab` so that number of api hits per minute does not get breached.
+3. For APIs which takes list of parameters as inputs, window functions like `collect_list` can be used
+   before `RestApiEnrich` Gem to reduce
+   the number of api calls.
+   :::
+
+:::note
+All input parameters are expected to be in string format. Other column types such as array/JSON/struct can be created
+using combination of aggregate/window Gems along with reformat component and then can be cast as string prior to paasing the column in `RestAPIEnrich Gem`
+as per requirement.
+:::
+
+### Example 1
+
+Let's try to fetch prices for few cryptocurrencies from [coin-api](https://www.coinapi.io/).
+
+We would be taking cryptocurrency and currency as input from DataFrame and pass url, headers as static values.
+Please note that URL in this example is created using static base url and adding cryptocurrency and currency as inputs
+from DataFrame.
+
+Also, we would be using Databricks-secrets to pass headers as it includes api-key.
+
+<div class="wistia_responsive_padding" style={{padding:'56.25% 0 0 0', position:'relative'}}>
+<div class="wistia_responsive_wrapper" style={{height:'100%',left:0,position:'absolute',top:0,width:'100%'}}>
+<iframe src="https://user-images.githubusercontent.com/103921419/184725747-88115fa5-b70b-4caf-b3e0-1f2476e15d6e.mp4" title="Rest API example 1" allow="autoplay;fullscreen" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" msallowfullscreen width="100%" height="100%"></iframe>
```

Comment:
In this video, the button is "Infer from cluster" rather than "infer schema"

however

the text recommends to select "infer schema"

Reply:
![image](https://user-images.githubusercontent.com/20563764/185037874-2cc908df-8919-43b9-9817-e39da02cc3a5.png)

Reply:
Will change the text in gem.